src/Workspaces/Core/MSBuild/MSBuild/DiagnosticReporter.cs

```diff
@@ -2,31 +2,31 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#nullable disable
-
 using System;
 using System.Collections.Immutable;
+using System.ComponentModel;
 using Microsoft.CodeAnalysis.MSBuild.Logging;
+using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.MSBuild
 {
     internal sealed class DiagnosticReporter
     {
-        internal ImmutableList<WorkspaceDiagnostic> Diagnostics;
+        internal ImmutableList<WorkspaceDiagnostic> _diagnostics;
```

Comment:
the name before was correct.  it is exposed outside the type, so should be pascal cased.  `_` is for privates.